lib/pdf_fill/forms/va686c674.rb

```diff
@@ -0,0 +1,1833 @@
+            },
+            'end_date' => {
+              'month' => {
+                key: 'veteran_marriage_history.end_date.month[%iterator%]',
+                limit: 2,
+                question_num: 14,
+                question_suffix: 'A',
+                question_text: 'PREVIOUS MARRIAGE HISTORY > TERMINATION MONTH'
+              }, # end of termination month
+              'day' => {
+                key: 'veteran_marriage_history.end_date.day[%iterator%]',
+                limit: 2,
+                question_num: 14,
+                question_suffix: 'B',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  TERMINATION DAY'
+              }, # end of termination day
+              'year' => {
+                key: 'veteran_marriage_history.end_date.year[%iterator%]',
+                limit: 4,
+                question_num: 14,
+                question_suffix: 'C',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  TERMINATION YEAR'
+              } # end of termination year
+            }, # end of end date
+            'end_location' => {
+              'city' => {
+                key: 'veteran.previousMarriage.terminationCity[%iterator%]',
+                limit: 18,
+                question_num: 14,
+                question_suffix: 'D',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  TERMINATION CITY'
+              }, # end city
+              'state' => {
+                key: 'veteran.previousMarriage.terminationState[%iterator%]',
+                limit: 2,
+                question_num: 14,
+                question_suffix: 'E',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  TERMINATION STATE'
+              } # end state
+            } # end of end_location
+          }, # end veteran_marriage_history
+          'spouse_marriage_history' => {
+            limit: 4,
+            first_key: 'full_name',
+            'full_name' => {
+              'first' => {
+                key: 'veteranSpouse.previousSpouse.firstName[%iterator%]',
+                limit: 12,
+                question_num: 15,
+                question_suffix: 'A',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  SPOUSES PREVIOUS SPOUSE FIRST NAME'
+              },
+              'middleInitial' => {
+                key: 'veteranSpouse.previousSpouse.middleInitial[%iterator%]',
+                limit: 1,
+                question_num: 15,
+                question_suffix: 'B',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  SPOUSES PREVIOUS SPOUSE MIDDLE INITIAL'
+              },
+              'last' => {
+                key: 'veteranSpouse.previousSpouse.lastName[%iterator%]',
+                limit: 18,
+                question_num: 15,
+                question_suffix: 'C',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  SPOUSES PREVIOUS SPOUSE LAST NAME'
+              },
+              'suffix' => {
+                # @TODO not in the form
```

Comment:
Need the TODO?

Reply:
See above note about comments - thanks :)